hw/ip/pwm/data/pwm.hjson

```diff
@@ -87,6 +87,37 @@
       desc: "End-to-end bus integrity scheme."
     }
   ]
+  inter_signal_list: [
+    { struct:  "racl_policy_vec",
+      type:    "uni",
+      name:    "racl_policies",
+      act:     "rcv",
+      package: "top_racl_pkg",
+      desc:    '''
+        Policy vector distributed to the subscribing RACL IPs.
```

Comment:
I wonder whether this text needs changing slightly. It looks a bit like PWM is the block that's distributing something (which is backwards). If I've got this right, PWM *is* a subscribing IP, so maybe something like "RACL policy vector, used by the IP to control register access permissions." ?